packages/now-static-build/src/index.ts

```diff
@@ -328,24 +392,30 @@ export async function build({
       );
     } else {
       if (meta.isDev) {
-        debug(`WARN: "${devScript}" script is missing from package.json`);
+        debug(`WARN: A dev script is missing.`);
         debug(
           'See the local development docs: https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/#local-development'
         );
       }
 
-      const buildScript = getCommand(pkg, 'build', config as Config);
-      debug(`Running "${buildScript}" script in "${entrypoint}"`);
-
-      const found = await runPackageJsonScript(
-        entrypointDir,
-        buildScript,
-        spawnOpts
+      const buildScript = pkg ? getCommand(pkg, 'build', config) : null;
+      debug(
+        `Running "${config.buildCommand ||
+          buildScript}" script in "${entrypoint}"`
       );
 
+      const found =
+        typeof config.buildCommand === 'string'
+          ? await execCommand(config.buildCommand, {
+              ...spawnOpts,
+              cwd: entrypointDir,
+            })
+          : await runPackageJsonScript(entrypointDir, buildScript!, spawnOpts);
```

Comment:
Similarly here:

```ts
const cmd = config.buildCommand || `yarn run ${devScript}`;
const found = await execCommand(cmd, {
  ...spawnOpts,
  cwd: entrypointDir,
})
```

Reply:
@styfle in this case I'd rather not change it, since it decides to use `npm` or `yarn` in `runPackageJsonScript`